terraform/eks/adot-operator/adot_collector_deployment.tpl

```diff
@@ -0,0 +1,11 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: aoc
+  namespace: ${AOC_NAMESPACE}
+spec:
+  image: public.ecr.aws/aws-observability/aws-otel-collector:latest
+  mode: deployment
```

Comment:
can we make the `mode` as variable? what if we need to add the test cases running in daemonset, etc.

Reply:
Yeah, this makes sense. I will make the change.